bonus/bonusAvailable.md

```diff
@@ -15,6 +15,7 @@
 
 ### bonus_available_v2.json
 - pointed by app > `1.10.0.4`
-
-
-
+- A new logic for handling a bonus visibility has been implemented. Now every bonus has a new property `visibility` which can be evaluated with 3 different values:
```

Comment:
Could you talk also about `hidden` property that will be deprecated?

Reply:
Added in 78dbe19